src/main/resource-applier.ts

```diff
@@ -64,19 +73,24 @@ export class ResourceApplier {
       resources.forEach((resource, index) => {
         fs.writeFileSync(path.join(tmpDir, `${index}.yaml`), resource);
       });
-      const cmd = `"${kubectlPath}" apply --kubeconfig "${proxyKubeconfigPath}" -o json -f "${tmpDir}"`;
+      kubectlArgs = kubectlArgs.concat(["-f", `"${tmpDir}"`]);
+      const cmd = `"${kubectlPath}" ${subCmd} --kubeconfig "${proxyKubeconfigPath}" ${kubectlArgs.join(" ")}`;
 
-      console.log("shooting manifests with:", cmd);
-      exec(cmd, (error, stdout, stderr) => {
+      logger.info(`[RESOURCE-APPLIER] running cmd ${cmd}`);
+      exec(cmd, (error, stdout) => {
         if (error) {
-          reject(`Error applying manifests:${error}`);
-        }
+          logger.error(`[RESOURCE-APPLIER] cmd errored: ${error}`);
+          const splittedError = error.toString().split(`.yaml": `);
```

Comment:
```suggestion
          const splitError = error.toString().split(`.yaml": `);
```
past tense of split is ... split  :)